docs/usage/training_time_compression/quantization_aware_training/Usage.md

```diff
@@ -0,0 +1,96 @@
+# Use Neural Network Compression Framework (NNCF) for Quantization Aware Training in PyTorch
```

Comment:
Use NNCF for Quantization Aware Training in PyTorch

Reply:
Done